src/curated-corpus/components/actions/DismissProspectAction/DismissProspectAction.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useDismissProspectMutation } from '../../../../api/generatedTypes';
+import { IconButton } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+import { useStyles } from './DismissProspectAction.styles';
+
+interface DismissProspectActionProps {
+  // sent by prospectListCard
+  prospectId: string;
+
+  // sent by prospecting page
+  onDismissProspect: (prospectId: string, errorMessage?: string) => void;
+}
+
+/**
+ * This component renders a cross (close) button and calls the dismissProspect mutation
+ * @param props
+ * @returns
+ */
+export const DismissProspectAction: React.FC<DismissProspectActionProps> = (
+  props
+) => {
+  const classes = useStyles();
+  const { prospectId, onDismissProspect } = props;
+
+  const [dismissProspect] = useDismissProspectMutation();
+
+  const onClick = async () => {
+    // call the dismissProspect mutation function and only extract the errors object
+    const { errors } = await dismissProspect({
+      variables: { id: prospectId },
+    });
```

Comment:
What's the motivation behind only extracting the errors object here?

Reply:
This function is called with the `prospectId` for the the prospect that has to be dismissed. And that `prospectId` passed in by the parent, so we don't really need to extract the `data` object since we don't really care about any data being returned here.